src/client/hud.cpp

```diff
@@ -226,13 +226,9 @@ void Hud::drawItem(const ItemStack &item, const core::rect<s32>& rect,
 //NOTE: selectitem = 0 -> no selected; selectitem 1-based
```

Comment:
While we are touching this function, please also tidy up this line:
```suggestion
// NOTE: selectitem = 0 -> no selected; selectitem is 1-based
```